md/codeBlock/fenced.ts

```diff
@@ -2,6 +2,13 @@ import { mostConsecutive } from "../../string/sequence.ts";
 import * as infoString from "./infoString.ts";
 import { FENCED_CODE_BLOCK_REGEX } from "./regex.ts";
 
+export type FencedCodeBlockDetails = infoString.Info & {
```

Comment:
Given that these types have the shapes of AST nodes, I'm tempted to use `Node` rather than `Details`. I'd like to have a full Markdown parser eventually with similar object shapes for each syntactic element, but they will probably contain more information than this such as positional information and may convey infoString pieces, fences, and code content as child nodes rather than as strings. So I'm using the word `Details` for now, and I can rename them later if they happen to conform to future AST node shapes.